.pre-commit-config.yaml

```diff
@@ -40,4 +40,6 @@ repos:
     rev: v1.10.0
     hooks:
       - id: mypy
-        additional_dependencies: [ types-all ]
+        files: clouds\/.*
+        additional_dependencies: [types-all]
+        args: [--ignore-missing-imports, --disallow-untyped-defs]
```

Comment:
can you remove this line `args: [--ignore-missing-imports, --disallow-untyped-defs]` and add:
```
[tool.mypy]
show_error_codes = true
warn_unused_ignores = true
check_untyped_defs = true
disallow_any_generics = true
disallow_incomplete_defs = true
disallow_untyped_defs = true
no_implicit_optional = true
```

to `pyproject.toml` and see if all work as expected?

Reply:
already exists in `pyproject.toml`.
checked and it works as excepted after removing `args: [--ignore-missing-imports, --disallow-untyped-defs]` (it reports missing typings)

Reply:
done